ansible/roles/install-logging/files/prometheus_datasource.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Prometheus",
+  "type": "prometheus",
+  "access": "proxy",
+  "url": "http://localhost:9090",
```

Comment:
The 9090 port is taken by Cockpit server, which we routinely use. Is it easy to change this and use other port, like 9999 for instance?

Reply:
I created USHIFT-577 to address this issue separately.

Reply:
Would people be installing the logging components on the devenv VM?

Reply:
I think so, otherwise, the benchmark collection is not complete. 
For R4E, we implemented a special option to embed the collection tool in the image.

Reply:
What do you mean the benchmark collection is not complete?
The prometheus config will scrape other nodes all the same.

Reply:
Moving the discussion to USHIFT-577 issue.